src/simulation.py

```diff
@@ -0,0 +1,77 @@
+"""This module provides functions to run the simulation using the configuration file or the simulation options object."""
+
+from pyseldon import seldoncore
+
+
+def run_simulation_from_config_file(
+    config_file_path, agent_file_path=None, network_file_path=None, output_dir_path=None
+):
+    """
+    Run the simulation using the configuration file.
+
+    Args:
+        config_file_path (str): The path to the configuration file.
+        agent_file_path (str): The path to the agent file.
+        network_file_path (str): The path to the network file.
+        output_dir_path (str): The path to the output file.
+    """
+    seldoncore.run_simulation(
+        config_file_path=config_file_path,
+        options=None,
+        agent_file_path=agent_file_path,
+        network_file_path=network_file_path,
+        output_dir_path=output_dir_path,
+    )
```

Comment:
This is a proxy call, and doesn't really add anything, the documentation can be in the `.cpp` file.